cric-predict/src/Components/LoginSignup.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import Header from './Header.js';
+import Form from './Form.js';
+import Image from './bg1.jpg';
+import {BrowserRouter as Router, Link, Route, Switch} from 'react-router-dom';
+
+function LoginSignup() {
```

Comment:
I'm a little confused about this hierarchy 3 different components are being used to render authentication elements